scipy/stats/tests/test_stats.py

```diff
@@ -4205,6 +4205,149 @@ def test_ttest_ind_with_uneq_var():
                      ([0, np.nan], [1, np.nan]))
 
 
+class TestTtestTrimmed(object):
+
+    def test_ttest_trimmed1(self):
```

Comment:
Where are these numbers coming from? If the original PR, was there a citation anywhere? If not, and if we can't find them, I think we need to replace them.

Reply:
The original author said that ["they came from R" ](https://github.com/scipy/scipy/pull/9678/files/27045ec45de313ecb8f6c735c271e9fcd211e1cf#r328988475) and provided a link to an R implementation (that we knew about), but those examples aren't there. So far I think they are just randomly generated. I also noticed that they are uncited but kept them to see if as I made changes I broke anything.

Reply:
That's fine. Keep them but check the results against that function in R, and write how that's done (e.g. "see `test_compare_r_1`") as a comment.